lib/phoenix/endpoint.ex

```diff
@@ -835,6 +835,9 @@ defmodule Phoenix.Endpoint do
     * `:compress` - whether to enable per message compresssion on
       all data frames, defaults to false
 
+    * `:subprotocol` - supported websocket subprotocol.
+      Used for handshake `Sec-WebSocket-Protocol` response header
```

Comment:
What is the default value? What is the type of this option? And as I said, let's go with `:subprotocols`.